django/db/migrations/exceptions.py

```diff
@@ -25,6 +25,13 @@ class CircularDependencyError(Exception):
     pass
 
 
+class InconsistentMigrationHistory(Exception):
+    """
+    Raised when a migration is found to be applied before its dependencies.
```

Comment:
I'd rephrase that to "Raised when an applied migration has some of its dependencies not applied."